backends/occa/ceed-occa-basis.c

```diff
@@ -13,11 +13,18 @@
 // the planning and preparation of a capable exascale ecosystem, including
 // software, applications, hardware, advanced system engineering and early
 // testbed platforms, in support of the nation's exascale computing imperative.
-
 #include "ceed-occa.h"
 
 // *****************************************************************************
-// * BASIS: Apply, Destroy, CreateTensorH1
+// Apply basis evaluation from nodes to quadrature points or vice-versa
+// @param basis Basis to evaluate
+// @param tmode \ref CEED_NOTRANSPOSE to evaluate from nodes to quadrature
+//     points, \ref CEED_TRANSPOSE to apply the transpose, mapping from
+//     quadrature points to nodes
+// @param emode \ref CEED_EVAL_INTERP to obtain interpolated values,
+//     \ref CEED_EVAL_GRAD to obtain gradients.
+// @param u input vector
+// @param v output vector
```

Comment:
I expect this isn't actually processed by Doxygen because it isn't `///`, though I don't think it should be because it isn't a public interface.